content/posts/2023/previewing-workspace-files.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Previewing workspace files"
+date: 2023-11-13T14:00:00+01:00
+---
+
+For a long time, the REANA web UI has allowed users to preview certain types of files present in the
+workflow workspace. This has proven to be very convenient for users who want to quickly glance through
+images and text files in their workspaces without having to download them. 
+Expanding on this, recent enhancements in REANA now include the previewing of PDF and ROOT files, 
+increasing the number of file types that can be inspected within the browser.
+
+<!--more-->
+
+## Images and text files preview
+
+Probably amongst the most common file types in a workflow workspace are images and text files. The web UI has
+always allowed to preview them from a modal window opened just by clicking on the name of the file in the workspace list.
+From there, you can also download the file if needed. Note that text files are not limited to `.txt` files, 
+but they also include logs, code, and other text-based files.
+
+![Text file preview](/images/text-file-preview.png)
+
+## HTML files preview
+
+Since REANA 0.8.1, users have been able to preview HTML files directly in the web UI.
```

Comment:
Again we can replace "past" by "present" and go like:

- The previewing of HTML files can be valuable especially for Snakemake users. The Snakemake workflow execution creates a detailed report in the HTML format. You can search your workspace for "report.html" and preview it in a new browser tab: